docs/cli/releases.mdx

```diff
@@ -118,13 +118,13 @@ sentry-cli releases set-commits "$VERSION" --auto --ignore-missing
 
 ## Managing Release Artifacts
 
-When you are working with JavaScript and other platforms, you can upload release artifacts to Sentry which are then considered during processing. The most common release artifact are [source maps](/platforms/javascript/sourcemaps/) for which `sentry-cli` has specific support.
+When you're working with JavaScript and other platforms, you can upload release artifacts (which are taken into account during processing) to Sentry. The `sentry-cli` has specific support for [source maps](/platforms/javascript/sourcemaps/), which are our most common release artifacts.
 
-To manage release artifacts the `sentry-cli releases files` command can be used which itself provides various sub commands.
+To manage release artifacts the `sentry-cli files` command can be used which itself provides various sub commands.
 
 ### Upload Files
 
-The most common use case is to upload files. For the generic upload the `sentry-cli releases files VERSION upload` command can be used. However since most release artifacts are JavaScript source map related we have a [Upload Source Maps](#sentry-cli-sourcemaps) convenience method for that.
+The most common use case is to upload files. For the generic upload the `sentry-cli files upload` command can be used. However since most release artifacts are JavaScript source map related we have a [Upload Source Maps](#sentry-cli-sourcemaps) convenience method for that.
```

Comment:
```suggestion
The most common use case is to upload files. For the generic upload, the `sentry-cli files upload` command can be used. However, since most release artifacts are JavaScript source map related, we have an [Upload Source Maps](/cli/releases/#upload-source-maps) convenience method for that.
```

Reply:
Included in https://github.com/getsentry/sentry-docs/pull/10288